app/models/qernel/node.rb

```diff
@@ -415,25 +415,25 @@ def calculate
   # @return [Float] The demand of this node
   #
   def update_demand
```

Comment:
Hi! This is very dangerous to do. Like this we will not know if the modelling was wrong or insufficient, the engine will just silently break and energy flows might not add up without us knowing. Please revert all changes in this file!